calendar_switzerland/migrations/10.0.1.0.1/pre-migration.py

```diff
@@ -0,0 +1,40 @@
+# -*- coding: utf-8 -*-
+##############################################################################
+#
+#    Copyright (C) 2018 Compassion CH (http://www.compassion.ch)
+#    Releasing children from poverty in Jesus' name
+#    @author: Emanuel Cino <ecino@compassion.ch>
+#
+#    The licence is in the file __manifest__.py
+#
+##############################################################################
+
+
+def migrate(cr, version):
+    if not version:
+        return
+
+    # Add "Campagne Marketing" tag to all calendar.event
+    cr.execute("WITH meeting_categories_to_insert as\
+                (\
+                    SELECT\
+                        ce.id as event_id,\
+                        (select id from calendar_event_type where name = 'Campagne Marketing') as type_id\
```

Comment:
Check travis errors. Line length should not go beyond 79 characters. For the query formating you could use a docstring
```python
cr.execute("""
this way you can
go at the line without need to put \ 
""")
```